module_3/src/service/User.js

```diff
@@ -0,0 +1,63 @@
+import API from './Api.js';
+import { userModel } from '../model/user.js';
+import { v4 as uuidv4 } from 'uuid';
+import statusCodes from '../../statusCodes.js';
+
+class User extends API {
+  constructor() {
+    super();
+    this.users = [];
+
+  }
+
+  findUser(id) {
```

Comment:
`findById` or `findOne`
`User` part is not necessary as it is already scoped by `User` class